backend/src/src-cvesearch/src/main/java/com/bosch/osmi/sw360/cvesearch/entitytranslation/EntityTranslator.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2015-2016.
+ * Part of the SW360 Portal Project.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package com.bosch.osmi.sw360.cvesearch.entitytranslation;
+
+import java.util.function.Function;
+
+public interface EntityTranslator<T, R> extends Function<T, R> {
```

Comment:
That is like an alias, which results in
- you have more information,
- it lets you use the IDE to find all EntityTranslators and
- more

Sadly the weak type system prevents you from getting some even better benefits.